charts/eks-operator/templates/deployment.yaml

```diff
@@ -27,7 +30,11 @@ spec:
           value: {{ .Values.noProxy }}
 {{- if .Values.additionalTrustedCAs }}
         volumeMounts:
-          - mountPath: /etc/ssl/certs/ca-additional.pem
+          - mountPath: /etc/rancher/ssl/ca-additional.pem
```

Comment:
In the Rancher PR you only added it to `/etc/pki/anchors....` and removed the custom Rancher dir. Why does eks still need the custom rancher dir ?

Reply:
Because the eks-operator user owns the `/etc/rancher/ssl` directory so `c_rehash` works there.

Reply:
Needs a comment in this file saying _"as opposed to rancher which runs `update-ca-certificates`, we run c_rehash and need both dirs...."_

Reply:
Done.